src/theory/arith/theory_arith_private.cpp

```diff
@@ -3478,6 +3478,23 @@ void TheoryArithPrivate::check(Theory::Effort effortLevel){
   }
   d_statistics.d_avgUnknownsInARow.addEntry(d_unknownsInARow);
 
+  if (options::useFC())
+  {
+    for (std::size_t i = 0; i < d_fcSimplex.getPivots(); ++i)
+    {
+      d_containing.d_out->spendResource(
+          ResourceManager::Resource::ArithPivotStep);
+    }
+  }
+  else
+  {
+    for (std::size_t i = 0; i < d_dualSimplex.getPivots(); ++i)
+    {
+      d_containing.d_out->spendResource(
+          ResourceManager::Resource::ArithPivotStep);
+    }
+  }
```

Comment:
How about:

```suggestion
  size_t nPivots = options::useFC() ? d_fcSimplex.getPivots() : d_dualSimplex.getPivots();
  for (std::size_t i = 0; i < d_fcSimplex.getPivots(); ++i)
  {
    d_containing.d_out->spendResource(
        ResourceManager::Resource::ArithPivotStep);
  }
```

I actually started thinking about it because the loops don't quite conform to our style guidelines for [loop conditions](https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#loop-conditions).